core/quality/mysterium_morqa.go

```diff
@@ -315,6 +316,32 @@ func (m *MysteriumMORQA) ProviderStatuses(providerID string) (node.MonitoringAge
 	return statuses, nil
 }
 
+// ProviderSessionsList fetch provider sessions list from quality oracle.
+func (m *MysteriumMORQA) ProviderSessionsList(providerID, rangeTime string) (node.MonitoringAgentSessions, error) {
+	id := identity.FromAddress(providerID)
+
+	request, err := requests.NewSignedGetRequest(m.baseURL, fmt.Sprintf("provider/sessions?range=%s", rangeTime), m.signer(id))
+	if err != nil {
+		return nil, err
+	}
+
+	response, err := m.client.Do(request)
+	if err != nil {
+		log.Err(err).Msg("Failed to request provider monitoring sessions list")
```

Comment:
This is not monitoring. 
```suggestion
		log.Err(err).Msg("Failed to request provider sessions list")
```